[2025-01-06T17:10:52.094+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T17:10:52.125+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.end_task manual__2025-01-06T17:10:33.683938+00:00 [queued]>
[2025-01-06T17:10:52.143+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.end_task manual__2025-01-06T17:10:33.683938+00:00 [queued]>
[2025-01-06T17:10:52.145+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T17:10:52.170+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): end_task> on 2025-01-06 17:10:33.683938+00:00
[2025-01-06T17:10:52.178+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=316) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T17:10:52.181+0000] {standard_task_runner.py:72} INFO - Started process 317 to run task
[2025-01-06T17:10:52.181+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'end_task', 'manual__2025-01-06T17:10:33.683938+00:00', '--job-id', '836', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmpq3efdqpr']
[2025-01-06T17:10:52.183+0000] {standard_task_runner.py:105} INFO - Job 836: Subtask end_task
[2025-01-06T17:10:52.258+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.end_task manual__2025-01-06T17:10:33.683938+00:00 [running]> on host 97bbb29a7501
[2025-01-06T17:10:52.418+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T17:10:33.683938+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T17:10:33.683938+00:00'
[2025-01-06T17:10:52.420+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T17:10:52.443+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-06T17:10:52.446+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here we end! "']
[2025-01-06T17:10:52.460+0000] {subprocess.py:99} INFO - Output:
[2025-01-06T17:10:52.462+0000] {subprocess.py:106} INFO - Here we end!
[2025-01-06T17:10:52.463+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-06T17:10:52.510+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T17:10:52.512+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=end_task, run_id=manual__2025-01-06T17:10:33.683938+00:00, execution_date=20250106T171033, start_date=20250106T171052, end_date=20250106T171052
[2025-01-06T17:10:52.559+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T17:10:52.588+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-06T17:10:52.593+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
